sacc/tracers/base.py

```diff
@@ -42,21 +49,18 @@ def __init__(self, name, **kwargs):
         self.quantity = quantity
         self.metadata = kwargs.pop('metadata', {})
 
-    def __init_subclass__(cls, tracer_type):
-        cls._tracer_classes[tracer_type.lower()] = cls
-        cls.tracer_type = tracer_type
 
     @classmethod
-    def make(cls, tracer_type, name, *args, **kwargs):
+    def make(cls, type_name, name, *args, **kwargs):
         """
-        Select a Tracer subclass based on tracer_type
+        Select a Tracer subclass based on type_name
         and instantiate in instance of it with the remaining
         arguments.
 
         Parameters
         ----------
-        tracer_type: str
-            Must correspond to the tracer_type of a subclass
+        type_name: str
+            Must correspond to the type_name of a subclass
```

Comment:
Shouldn't the type of the tracer base class always be a tracer? Why is it agnostic?